### ISSUE TITLE
Unable to run after v3.1.0 upgrade

### ISSUE THREAD

**Issue by SSoft7** [FlareSolverr/FlareSolverr#743]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: v3.1.0
- Last working FlareSolverr version: v3.0.4
- Operating system: Debian
- Are you using Docker: yes
- FlareSolverr User-Agent (see log traces or / endpoint): NA
- Are you using a VPN: no
- Are you using a Proxy: no
- Are you using Captcha Solver: no
- If using captcha solver, which one:
- URL to test this issue: N/A
```

### Description

Getting TypeError: 'type' object is not subscriptable after upgrading to v3.1.0

### Logged Error Messages

```text
2023-03-24 17:35:23 INFO     Chrome / Chromium path: /usr/bin/chromium
Traceback (most recent call last):
  File "/app/flaresolverr/src/flaresolverr.py", line 11, in <module>
    from dtos import V1RequestBase
  File "/app/flaresolverr/src/dtos.py", line 51, in <module>
    class V1ResponseBase(object):
  File "/app/flaresolverr/src/dtos.py", line 56, in V1ResponseBase
    sessions: list[str] = None
TypeError: 'type' object is not subscriptable
Traceback (most recent call last):
  File "/app/flaresolverr/src/flaresolverr.py", line 11, in <module>
    from dtos import V1RequestBase
  File "/app/flaresolverr/src/dtos.py", line 51, in <module>
    class V1ResponseBase(object):
  File "/app/flaresolverr/src/dtos.py", line 56, in V1ResponseBase
    sessions: list[str] = None
TypeError: 'type' object is not subscriptable
```

**Comment by ilike2burnthing** [comment]
There is no v3.1.1 so what are you running?

Provide the output from `uname -a` on your host.

**Comment by SSoft7** [comment]
> There is no v3.1.1 so what are you running?
> 
> Provide the output from `uname -a` on your host.

My mistake. It's v3.1.0

Uname -a from the container
`Linux 98264831cf18 5.4.0-139-generic #156-Ubuntu SMP Fri Jan 20 17:27:18 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by ilike2burnthing** [comment]
> on your host

**Comment by ngosang** [comment]
The only reason for this error is you are using Python < 3.9. That is really realy stragne since we include Python 3.11 in the Docker image.
https://stackoverflow.com/a/63460173

Could you run ``/usr/local/bin/python`` in the container to see the Python version?

**Comment by SSoft7** [comment]
> The only reason for this error is you are using Python < 3.9. That is really realy stragne since we include Python 3.11 in the Docker image. https://stackoverflow.com/a/63460173
> 
> Could you run `/usr/local/bin/python` in the container to see the Python version?

Thanks for pointing out...I was using my own image by copying your dockerfile logic...but the base was different and so was the python version.

Closing this as it is resolved now.

**Comment by ngosang** [comment]
I included some checks ae18559db148795ab18fcbdcaacee9838884a17e
